libs/cdk/src/lib/utils/directives/focusable-item/focusable-item.directive.ts

```diff
@@ -92,27 +92,49 @@ export class FocusableItemDirective implements HasElementRef {
         return this._tabbable ? 0 : -1;
     }
 
+    /** Element reference. */
+    public readonly elementRef = inject<ElementRef<HTMLElement>>(ElementRef);
+    /** @hidden */
+    private readonly _focusableObserver = inject(FocusableObserver);
+    /** @hidden */
+    protected readonly _destroy$ = inject(DestroyedService);
+    /** @hidden */
+    private readonly _tabbableElementService = inject(TabbableElementService);
+    /** @hidden */
+    private readonly _liveAnnouncer = inject(LiveAnnouncer);
     /** @hidden */
-    constructor(
-        private _elementRef: ElementRef<HTMLElement>,
-        private _focusableObserver: FocusableObserver,
-        private _destroy$: DestroyedService,
-        private _tabbableElementService: TabbableElementService,
-        private _liveAnnouncer: LiveAnnouncer
-    ) {
+    protected readonly _zone = inject(NgZone);
+
+    /** @hidden */
+    constructor() {
         this._focusableObserver
-            .observe(this._elementRef, false)
+            .observe(this.elementRef, false)
```

Comment:
What is the reason we are changing from private to public? Maybe comment would help .